src/screens/SendTokens.js

```diff
@@ -126,25 +154,20 @@ class SendTokens extends React.Component {
   }
 
   /**
-   * Get inputs and outputs data from child components  
+   * Get inputs and outputs data from child components
    * Each child component holds inputs/outputs for one token
    *
    * @return {Object} Object holding all inputs and outputs {'inputs': [...], 'outputs': [...]}
    */
   getData = () => {
-    let data = {'inputs': [], 'outputs': []};
+    let data = {'inputs': [], 'outputs': [], 'tokens': []};
     for (const ref of this.references) {
       const instance = ref.current;
       let dataOne = instance.getData();
       if (!dataOne) return;
-      if (hathorLib.wallet.isHardwareWallet()) {
-        dataOne = instance.validateInputsAndOutputs(dataOne);
-        if (!dataOne) return;
-        // currently we only support HTR
-        data['tokens'] = [];
-      }
       data['inputs'] = [...data['inputs'], ...dataOne['inputs']];
       data['outputs'] = [...data['outputs'], ...dataOne['outputs']];
+      data['tokens'] = [...data['tokens'], ...dataOne['tokens']];
```

Comment:
`getData` method does not add any key `'tokens'` to the returned object. I don't think we should have this line here, the tokens array is handled in the lib with the token data.

Reply:
Removed